unison-src/builtin-tests/interpreter-tests.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 set -ex
 
-ucm=$(stack exec -- which unison)
+ucm=$(cabal exec -- which unison)
```

Comment:
why change this?

Reply:
No, that was an accident. I'll fix it in a PR for the hashing change.